apps/vscode.py

```diff
@@ -0,0 +1,87 @@
+from talon.voice import Context, Key, press, Str
+from user.utils import parse_words_as_integer
```

Comment:
these should be relative, so that other people can put this code in some other path if they want to. `from ..utils import`

Reply:
also this line isn't necessary since parse_words_as_integer is already imported below

Reply:
Done